src/server/js/resources/health/schema.js

```diff
@@ -0,0 +1,26 @@
+import HealthModel from './model';
+
+const typeDefs = `
+    type HealthModel {
+        id: ID!
+        checkedOn: String
+    }
+
+    type Query {
+        healthChecks: [HealthModel]
+        health(id: ID!): HealthModel
+    }
```

Comment:
each non-scalar(`String/Number/...`) field in a query needs a resolver (see below)